scripts/src/actions/add-gui-workflows.ts

```diff
@@ -0,0 +1,41 @@
+import 'reflect-metadata'
+import * as terraform from '../terraform'
+import * as yaml from '../yaml'
+import YAML from 'yaml'
+
+// TODO: run this script as part of some workflow (PR?, sync?), for now run:
+//       npm run build && TF_WORKSPACE=ipfs node lib/actions/add-gui-workflows.js
+async function run(): Promise<void> {
+  const config = yaml.getConfig()
+
+  // TODO: we need a better abstraction for things that are defined in the YAML
+  //       ideally, we should be able to replace all of this with something like:
+  //       config.getRepositories().filter(repository => repository.topics.includes('ipfs-desktop')).map(repository => repository.name)
+  const repositories = config.getResources([terraform.GithubRepository]).filter(repository => {
+    return ((repository.value as YAML.Pair).value as YAML.YAMLMap).toJSON().topics?.includes('ipfs-desktop')
```

Comment:
we should change `((repository.value as YAML.Pair).value as YAML.YAMLMap).toJSON().topics?.includes('ipfs-desktop')` to something like

```ts
const GUI_TOPICS = ['ipfs-gui', 'gui']
const repoObj = ((repository.value as YAML.Pair).value as YAML.YAMLMap).toJSON()
const topics = repoObj.topics ?? []

return GUI_TOPICS.find((t) => topics.includes(t)) == null

Reply:
Better abstractions where we could do something like `config.getResources(GitHubRepository).filter(r => GUI_TOPICS.find(t => r.topics.includes(t))` are coming shortly with https://github.com/protocol/github-mgmt-template/pull/61

I'll rebase this once the PR I mentioned lands